File: Daily Challenge/Java/Flowchart Logic - 059 9.10.2025.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num1 = sc.nextInt(), num2 = sc.nextInt();
        int a = num1, b = num2;
        while(a > 0) {
            b = b * 10 + (a % 10);
            a /= 10;
        }
        int sum1 = b;
        a = num1; b = sum1;
        while(b > 0) {
            a = a * 10 + (b % 10);
            b /= 10;
        }
        sum1 += a;
        System.out.println(sum1);
    }
}

File: Daily Challenge/C/Flowchart Logic - 059 9.10.2025.c
#include <stdio.h>
int main() {
    int num1, num2;
    scanf("%d %d", &num1, &num2);
    while(num1 > 0) {
        num2 = num2 * 10 + (num1 % 10);
        num1 /= 10;
    }
    int sum1 = num2;
    while(num2 > 0) {
        num1 = num1 * 10 + (num2 % 10);
        num2 /= 10;
    }
    sum1 += num1;
    printf("%d
", sum1);
    return 0;
}